dask/array/core.py

```diff
@@ -3187,6 +3224,16 @@ def cov(m, y=None, rowvar=1, bias=0, ddof=None):
         return (dot(X, X.T.conj()) / fact).squeeze()
 
 
+@wraps(np.corrcoef)
+def corrcoef(x, y=None, rowvar=1):
+    c = cov(x, y, rowvar)
+    if c.shape == ():
+        return c/c
+    d = diag(c)
+    d = d.reshape((d.shape[0], 1))
+    return c / sqrt(d * d.T)
```

Comment:
I wonder if we should change the order of operations here.  Does the number of tasks or the need to coordinate intermediate results change significantly if we do something like `c / sqrt(d[None, :]) / sqrt(d[:, None])`?  (this is almost certainly not correct, but might not be far off.